site/content/kapp-controller/docs/v0.32.0/walkthrough.md

```diff
@@ -3,203 +3,241 @@ aliases: [/kapp-controller/docs/latest/walkthrough]
-  5:26:30PM: ok: delete app/simple-app (kappctrl.k14s.io/v1alpha1) namespace: default
-  5:26:30PM: ---- applying complete [2/2 done] ----
-  5:26:30PM: ---- waiting complete [2/2 done] ----
-
-  Succeeded
-  ```
+This walkthrough demonstrates how to install an example application on Kubernetes with kapp-controller. The example application is an HTTP server. You will use `examples/simple-app-git` directory for the YAML configuration.
+
+You can use [kapp](/kapp) or another tool such as kubectl to deploy the following YAML examples:
+
+1. [Install](install.md) kapp-controller onto your cluster.
+
+1. Install [examples/default-ns-rbac.yml](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/examples/rbac/default-ns.yml).
+
+    It creates `default-ns-sa` service account to change resources within the `default` namespace. The App CR in the next step uses the service account.
+
+    ```bash-plain
+    $ kapp deploy -a default-ns-rbac -f https://raw.githubusercontent.com/vmware-tanzu/carvel-kapp-controller/develop/examples/rbac/default-ns.yml
+    ```
+
+1. Install [examples/simple-app-git/1.yml](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/examples/simple-app-git/1.yml) App CR
+by running one of the following commands:
+
+    It specifies how to fetch, template, and deploy the example application.
+
+    ```bash-plain
+    $ kapp deploy -a simple-app -f https://raw.githubusercontent.com/k14s/kapp-controller/develop/examples/simple-app-git/1.yml
+    ```
+
+    ```bash-plain
+    $ kubectl apply -f https://raw.githubusercontent.com/k14s/kapp-controller/develop/examples/simple-app-git/1.yml
+    ```
+
+    Expect to see the following output:
+
+    ```bash-plain
+    Changes
+
+    Namespace  Name        Kind  Conds.  Age  Op      Wait to    Rs  Ri
+    default    simple-app  App   -       -    create  reconcile  -   -
+
+    Op:      1 create, 0 delete, 0 update, 0 noop
+    Wait to: 1 reconcile, 0 delete, 0 noop
+
+    Continue? [yN]: y
+
+    5:20:27PM: ---- applying 1 changes [0/1 done] ----
+    5:20:27PM: create app/simple-app (kappctrl.k14s.io/v1alpha1) namespace: default
+    5:20:27PM: ---- waiting on 1 changes [0/1 done] ----
+    5:20:27PM: ongoing: reconcile app/simple-app (kappctrl.k14s.io/v1alpha1) namespace: default
+    5:20:33PM: ok: reconcile app/simple-app (kappctrl.k14s.io/v1alpha1) namespace: default
+    5:20:33PM: ---- applying complete [1/1 done] ----
+    5:20:33PM: ---- waiting complete [1/1 done] ----
+
+    Succeeded
+    ```
+
+1. Run `kubectl get app` to verify that the app is deployed.
+
+1. Verify the status of the App CR by running one of the following commands:
+
+    **Note:** As of kapp-controller v0.31.0, inspect is deactivated by default. See [App CR spec](app-spec.md) for more details.
+
+    ```bash-plain
+    $ kapp inspect -a simple-app --status
+    ```
+
+    ```bash-plain
+    $ kubectl get app simple-app -oyaml
+    ```
```

Comment:
Thanks, @cppforlife. Would it make sense to remove all the `kubectl` commands since we are promoting `kapp` in the docs?

Reply:
@cppforlife  I disagree here --  a walkthrough is a beginner-level doc, so imo it's helpful in this context to have some "rosetta stone" style documentation showing equivalent kapp/kubectl commands for dealing with kapp-controller apps. Helps make us beginner friendly, backs up the assertion that we interop with other things, pulls back the curtain just a tiny bit.

Reply:
in that case, its confusing that it says expect to see output after kubectl since the output is from kapp. if you think we should keep kubectl, that's fine, but then i would recommend arranging it differently.

Reply:
Thanks @joe-kimmel-vmw and @cppforlife. I was not aware the output only applies to `kapp`. I will try to come up with a new solution to address this issue.